examples/src/Viewer.tsx

```diff
@@ -45,10 +46,10 @@ import {
     UI_TEMPLATE_URL_ROOT,
 } from "./api-settings";
 import ConversionForm from "./ConversionForm";
-import MetaballSimulator from "./MetaballSimulator";
-import { TrajectoryType } from "../src/constants";
+import MetaballSimulator from "./simulators/MetaballSimulator";
+import { TrajectoryType } from "../../src/constants";
```

Comment:
nit/cleanup: I suggest moving this line up closer to the other imports from ../../src